app/build.gradle

```diff
@@ -9,8 +9,8 @@ android {
         applicationId "org.literacyapp.synchronization"
         minSdkVersion 21
         targetSdkVersion 23
-        versionCode 1000000
-        versionName "1.0.0"
+        versionCode 8000
+        versionName "0.8.0.00"
```

Comment:
@elich11 Why do you use `0.8.0.00` as versionName? Please note that the versionCode has to be larger than the previous versionCode. If not, uploading the APK to the website will fail.